core/node/db_pruner/README.md

```diff
@@ -1,4 +1,4 @@
-# Db pruner
+# `zksync_node_db_pruner`
```

Comment:
why do we change style  here?
It's a `.md` file and not code docs, so it looks weird to me, or I miss smth?

Reply:
Just a matter of style. I think given that we have a lot of crate names, it'll be simpler to just use crate names as headers in these files rather than attempt to invent something unique in each case.

Generally, I think it's not that important, at least for the first iteration.